### PR TITLE
Security certs disa-stig

### PR DESCRIPTION
**Pull request by carkod**
## Done

Copy updates

## QA

Check https://ubuntu-com-10029.demos.haus/security/disa-stig against [copy docs](https://docs.google.com/document/d/1zJwZzc-cERj9YKNXFtmrXwynJptyT7-D3qCNzBctRfo/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4229

### CODE REVIEW

File: templates/security/disa-stig.html [hunk 1]
```diff
@@ -41,14 +41,14 @@ <h2>What is DISA-STIG?</h2>
...
-    <p class="u-sv3">DISA has, in conjunction with Canonical, developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
+    <p class="u-sv3">Together with Canonical DISA has, developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
```
Comment (bethcollins92) [inline templates/security/disa-stig.html:44]:
```suggestion
    <p class="u-sv3">Together with Canonical DISA has developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
```

**Review by bethcollins92 — APPROVED** [review]
LGTM, just a sneaky comma again

File: templates/security/disa-stig.html (hunk 1 above)
Comment (carkod) [reply templates/security/disa-stig.html:44]: mmm I think your suggestion is removing the comma. The PR has the comma in it, it's in green.

Comment (carkod) [reply templates/security/disa-stig.html:44]: Oh wait, I see, it should be moved after Canonical

**Pull request merged**